### ISSUE TITLE
can not Access cf domain ? 

### ISSUE THREAD

**Issue by H1-N1** [zizifn/edgetunnel#215]

**Comment by YixiaoOrg** [comment]
Yes, the worker domain is blocked(GFW). You can use your own domain.

**Comment by H1-N1** [comment]
> 



> Yes, the worker domain is blocked(GFW). You can use your own domain.

im not from China so it not about (GFW) 
all website powered by cf (cloudflar) can not be opened.

**Comment by YixiaoOrg** [comment]
> > 
> 
> > Yes, the worker domain is blocked(GFW). You can use your own domain.
> 
> im not from China so it not about (GFW) all website powered by cf (cloudflar) can not be opened.

![image](https://github.com/zizifn/edgetunnel/assets/86191112/28013cdf-cbe4-49bc-b5ad-ae65109d2b10)
Yes, currently I don't have a good solution for this problem.

https://github.com/zizifn/edgetunnel/issues/162#issuecomment-1575096833
